src/svc/invocable-scripts/schemas/ScriptLanguage.yml

```diff
@@ -1,2 +1,2 @@
 type: string
-enum: [flux, sql]
+enum: [flux, sql, influxQL]
```

Comment:
Just a question -- Why the capitalization in `influxQL`? The others are all lowercase. Could it just be `influxql`?

Reply:
Good question! It was originally defined in UI https://github.com/influxdata/ui/blob/9c77358d8d75e2c7bff59203db4324471d93506a/src/dataExplorer/components/resources/index.ts#L10, maybe I can change it to all lowercase to keep the consistency

Reply:
@sanderson updated in https://github.com/influxdata/openapi/pull/639/commits/59252bfe55885eb63d410d08156519915bac77ab